app/views/admin/loans/timeline/_list_header.html.slim

```diff
@@ -18,15 +18,15 @@ header.timeline-header
           li: a id="check-completed-ctrl" = t(:completed)
           li: a id="check-incomplete-ctrl" = t(:incomplete)
 
-        input type="hidden" name="step-ids" id="step-ids"
+        input.step-ids type="hidden" name="step-ids"
 
       .btn-group role="group" aria-label="Batch Actions"
         button.btn.btn-default.dropdown-toggle type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"
           => t("loan.batch_actions")
           span.caret
           span.sr-only = t(:dropdown_toggle)
 
-        = render "admin/loans/timeline/batch_actions"
+        = render "admin/loans/timeline/batch_actions", locals: {modal_id: 'adjust-dates-modal-timeline-list'}
```

Comment:
Can you please add comments to all places where the `modal_id` local appears saying that we can get rid of it once old timeline is removed.